src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs

```diff
@@ -12,6 +12,7 @@ public static class FrameworkConstants
         public static readonly Version MaxVersion = new Version(int.MaxValue, 0, 0, 0);
         public static readonly Version Version5 = new Version(5, 0, 0, 0);
         public static readonly Version Version6 = new Version(6, 0, 0, 0);
+        public static readonly Version Version7 = new Version(7, 0, 0, 0);
```

Comment:
This is a public API change and as such it required an Unshipped updates check. 

The CI will fail because this new API is not declared.